packages/plugin-align/src/Align.ts

```diff
@@ -62,7 +62,7 @@ export class Align<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T
      * @param [type]
      */
     static isAligned(node: VNode, type?: AlignType): boolean {
-        if (typeof node.modifiers.find(Attributes)?.get('style') !== 'string') {
+        if (typeof node.modifiers.get(Attributes).get('style') !== 'string') {
```

Comment:
Isn't it weird to use `get` in an `if` ? Because if there was no `Attributes` then you just created one. The usecase of `get` is for when you want to set a modifier while reusing the existing one *if* there is one. If you're not going to set the modifier then I would have kept `find`. This applies to all places where you do not actually do something with the retrieved modifier.